aspnet/web-forms/overview/security/create-a-secure-aspnet-web-forms-app-with-user-registration-email-confirmation-and-password-reset.md

```diff
@@ -32,7 +32,7 @@ This tutorial guides you through the steps required to create an ASP.NET Web For
 <a id="createWebForms"></a>
 ## Create an ASP.NET Web Forms App
 
-Start by installing and running [Visual Studio Express 2013 for Web](https://go.microsoft.com/fwlink/?LinkId=299058) or [Visual Studio 2013](https://go.microsoft.com/fwlink/?LinkId=306566). Install [Visual Studio 2013 Update 3](https://go.microsoft.com/fwlink/?LinkId=390465) or higher as well.
+Start by installing and running <!--[Visual Studio Express 2013 for Web](https://go.microsoft.com/fwlink/?LinkId=299058) or --> [Visual Studio 2013](https://go.microsoft.com/fwlink/?LinkId=306566). Install [Visual Studio 2013 Update 3](https://go.microsoft.com/fwlink/?LinkId=390465) or higher as well.
```

Comment:
JP, the two fwlinks you kept are both redirecting to a landing page that promotes the latest version. I think you should link instead to https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2013-and-other-products.

```suggestion
Start by installing and running Visual Studio Express 2013 for Web or Visual Studio 2013, via this Visual Studio [downloads page](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2013-and-other-products)
```